starport/services/network/networkchain/prepare.go

```diff
@@ -19,6 +19,21 @@ import (
 	"github.com/tendermint/starport/starport/services/network/networktypes"
 )
 
+// ResetGenesisTime reset the chain genesis time
+func (c Chain) ResetGenesisTime() error {
+	// set the genesis time for the chain
+	genesisPath, err := c.GenesisPath()
+	if err != nil {
+		return errors.Wrap(err, "genesis of the blockchain can't be read")
+	}
+	if err := cosmosutil.SetGenesisTime(genesisPath, 0); err != nil {
+		return errors.Wrap(err, "genesis time can't be set")
+	}
+
+	fmt.Println("\nGenesis time was reset")
```

Comment:
We shouldn't print to stdout in the services/network packages. Can we move this to cmd?